src/main/resources/db/migration/V3_04__insert_data_into_application_no_order_table.sql

```diff
@@ -0,0 +1,41 @@
+CREATE TEMP TABLE tmp_not_reserved_applications(application_reference text UNIQUE NOT NULL);
+
+-- create temp table to store not reserved applications
+INSERT INTO tmp_not_reserved_applications(application_reference)
```

Comment:
this might not be needed if done during after hours.

Reply:
its used for historic data not current data